src/devices/flex.c

```diff
@@ -387,7 +263,11 @@ r_device *flex_create_device(char *spec)
         fprintf(stderr, "Bad flex spec, missing name!\n");
         usage();
     }
+#ifdef WIN32
+    params->name = _strdup(c);
+#else
     params->name = strdup(c);
+#endif
     snprintf(dev->name, sizeof(dev->name), "General purpose decoder '%s'", c);
```

Comment:
Riddling the code with #ifdefs does not help maintenance. I'd rather have one file that contains all the platform defines.